sale_product_seasonality/models/sale_order.py

```diff
@@ -24,6 +24,13 @@ class SaleOrder(models.Model):
         column2="product_id",
         compute="_compute_season_allowed_product_ids",
     )
+    season_allowed_product_template_ids = fields.Many2many(
```

Comment:
this makes the other field useless because we could use the template as domain for variants.

Reply:
You're right.
My way was totally bad.
Redo in a very more easy way.